### DISCUSSION TITLE
Lab Assignment #1

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#4] [Assignments]
## Due 4/6 before class 

Please use this thread to post your [lab assignment's #1](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week1/lab_assignment/) link, which is the GitHub pages URL.

For example:
https://albertkun.github.io/22s-asiaam-191a-sample/week1/index.html

**Comment by lindseymardona** [comment]
https://lindseymardona.github.io/22s-asiaam-191a-labs/week1/index.html

**Comment by daikis12** [comment]
https://daikis12.github.io/22s-191a-assignments/week1/index.html

**Comment by stefuntran** [comment]
https://stefuntran.github.io/22s-asiaam-191a-labs/week%201/index.html

**Comment by ashley-lu** [comment]
https://ashley-lu.github.io/s22-asiaam-191a-assignments/week1lab/index.html

**Comment by angelineeow** [comment]
https://angelineeow.github.io/22s-asiaam-191a-labs/week1/index.html

**Comment by joonyloony** [comment]
https://joonyloony.github.io/22s-191a-assignments/week1/index.html

**Comment by willa-21** [comment]
https://willakurland.github.io/aa191-labs/Week-1/lab1.html

**Comment by ntran27** [comment]
https://ntran27.github.io/22s-asiaam-191a-labs/week1/index.html

**Comment by codingdianat** [comment]
https://codingdianat.github.io/aa191-assignments/week1/index.html

**Comment by NishuKrish** [comment]
https://nishukrish.github.io/AsianAm191A-Nish/week1/index.html

**Comment by kris10fu** [comment]
https://kris10fu.github.io/22s-asiaam-191a-labs/week1/index.html

**Comment by ananyakodali17** [comment]
https://ananyakodali17.github.io/asian-am-191a-labs/week1/index.html

**Comment by shubhgupta2018** [comment]
https://shubhgupta2018.github.io/22s-191a-assignments/week-1/index.html

**Comment by hajovel** [comment]
https://hajovel.github.io/211S-191A-Assignments/week1/index.html

**Comment by antonyblin** [comment]
https://antonyblin.github.io/aas191a-assignments/week1/index.html

**Reply by albertkun** [reply]
OH NO! Get some sleep! :(

**Comment by bittertang** [comment]
https://bittertang.github.io/22s-asiaam-191a-labs/week%201/index.html

**Comment by AHatalski** [comment]
https://github.com/AHatalski/ASAM191a_labs/blob/master/week%201/index.html

**Comment by penelopelamb** [comment]
https://penelopelamb.github.io/22s-aas-191a-labs/week1/index.html

**Comment by XiongHer** [comment]
https://xiongher.github.io/22s-asiaam-191a-labs/week1/index.html

**Comment by kericsp** [comment]
https://kericsp.github.io/aa191-labs/week1/index.html

**Comment by spolcharoen** [comment]
https://spolcharoen.github.io/22s-asiaam-191a-labs/week1/index.html

**Comment by kaittli** [comment]
https://kaittli.github.io/22s-asiaam-191a-labs/week%201/index.html

**Comment by emmashanson** [comment]
https://emmashanson.github.io/22s-asianam-191a-labs/week%201/index.html
